DigitalLearningSolutions.Web.Tests/Controllers/TrackingSystem/Delegates/DelegateCoursesControllerTests.cs

```diff
@@ -0,0 +1,207 @@
+﻿namespace DigitalLearningSolutions.Web.Tests.Controllers.TrackingSystem.Delegates
+{
+    using System.Collections.Generic;
+    using DigitalLearningSolutions.Data.DataServices;
+    using DigitalLearningSolutions.Data.Models.Common;
+    using DigitalLearningSolutions.Data.Models.Courses;
+    using DigitalLearningSolutions.Data.Services;
+    using DigitalLearningSolutions.Web.Controllers.TrackingSystem.Delegates;
+    using DigitalLearningSolutions.Web.Tests.ControllerHelpers;
+    using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.DelegateCourses;
+    using FakeItEasy;
+    using FluentAssertions;
+    using FluentAssertions.Execution;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+    using NUnit.Framework;
+
+    public class DelegateCoursesControllerTests
+    {
+        private readonly List<ApplicationDetails> applicationOptions = new List<ApplicationDetails>
```

Comment:
Our policy is to use NBuilder to set up test data unless using the constructor is significantly more concise.
It'll populate class members of basic types automatically.
E.g. Builder<ApplicationDetails>.CreateListOfSize(1).Build();
(Gives you an IList instead of a List, but shouldn't be a problem in general)